src/models/session/index.ts

```diff
@@ -1,2 +1 @@
 export * from "./logic";
-export * from "./session-state";
```

Comment:
SessionState is still experimental at this stage so I'm not exporting it yet.

Reply:
I'm still working my way through this PR but wanted to flag early that this (merged) PR is importing `StaticSessionState` https://github.com/theopensystemslab/planx-core/pull/35/files

This hasn't gone to prod yet, and I haven't yet informed BOPS that section names have been added to the payload so it's fine if we need to revert this and find another solution for this one - just let me know.

Reply:
I'll submit a patch to deprecate `StaticSessionState` usages in planx-new before merging this if that's okay. That means we can play around a little more with the interface and structure here without impacting anything for the time-being. I hope to get this in a production ready state soon though.